README.md

```diff
@@ -22,6 +22,13 @@ See the **[Quickstart guide](https://www.cloudquery.io/docs/quickstart)** for in
 - **Eliminate data silos**: Eliminate data silos across your organization, unifying data between security, infrastructure, marketing and finance teams.
 - **Unlimited scale**: CloudQuery plugins are stateless and can be scaled horizontally on any platform, such as EC2, Kubernetes, batch jobs or any other compute.
 
+## Use Cases
+
+- **Cloud Security Posture Management**: Use as an open source CSPM solution to monitor and enforce security policies across your cloud infrastructure for AWS, GCP, Azure and many more.
```

Comment:
Do we have some blog posts to link to these?

Reply:
Yes, though maybe we can add them to the links section?

Reply:
(I don't think we have ones specific to AWS, GCP and Azure if that's what you mean)